domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/directionsquestionnaire/PilotCourt.java

```diff
@@ -18,7 +19,10 @@ public String getName() {
     }
 
     public static boolean isPilotCourt(String courtName) {
+        if (courtName == null) {
+            return false;
+        }
         return Arrays.stream(PilotCourt.values())
-            .anyMatch(val -> val.name().equalsIgnoreCase(courtName));
+            .anyMatch(pilotCourt -> courtName.toLowerCase().contains(pilotCourt.name));
```

Comment:
is contains ok? should it be equals? what if we have something like north birmingham.

Reply:
That's exactly what the new logic is. 
```
It is accepted as pilot court if any of the below text is there in the suggested court name

- Manchestor
- Edmonton
- Birmingham
- Clerkenwell 
- Shoreditch

```

Reply:
cool 👍 😄